### PR TITLE
Change validator password

### LINKED ISSUE #5861: New command to change a validator's password
# 🚀 Feature Request

### Description

I had a discussion on Discord (https://discordapp.com/channels/476244492043812875/560308336248225812/710615612203728996) recently about #5836. Someone had an issue that could be resolved by implementing a command that allows changing the validator's password. I believe it should be easy to implement - in fact the code in the linked PR already does this at some point. I am happy to submit another PR if this functionality would be useful.

### Describe the solution you'd like

Create a command `validator accounts change-password` that asks for the old password and for a new password. As a result the key files are encrypted with the new password.

### Describe alternatives you've considered

A workaround will be possible by using the `validator accounts merge` command from the linked PR, The user could move the key files to some other directory and merge them back to the desired directory, changing the password in the meantime. A true workaround :wink:

### PR DESCRIPTION
**Pull request by rkapka**
**What type of PR is this?**

Feature

**What does this PR do? Why is it needed?**

Allows users to change their validator password by using a `validator accounts change-password` command.
This PR fixes problems that arise when running the validator client and:

-  multiple validators are stored in the same key store path 
-  their keys have been encrypted using different passwords

**Which issues(s) does this PR fix?**

Fixes #5861

### CODE REVIEW

File: validator/accounts/account.go [hunk 1]
```diff
@@ -214,6 +213,37 @@ func HandleEmptyFlags(cliCtx *cli.Context, confirmPassword bool) (string, string
 	return path, passphrase, nil
 }
 
+// ChangePassword changes the password for all keys located in a keystore.
+// Password is changed only for keys that can be decrypted using the old password.
+func ChangePassword(keystorePath string, oldPassword string, newPassword string) error {
+	err := changePasswordForKeyType(keystorePath, params.BeaconConfig().ValidatorPrivkeyFileName, oldPassword, newPassword)
```
Comment (rauljordan) [inline validator/accounts/account.go:219]: Line too long, split into
changePasswordForKeyType(
  keystorePath,
  ...,
)

File: validator/accounts/account.go [hunk 2]
```diff
@@ -214,6 +213,37 @@ func HandleEmptyFlags(cliCtx *cli.Context, confirmPassword bool) (string, string
...
+	if err != nil {
+		return err
+	}
+	err = changePasswordForKeyType(keystorePath, params.BeaconConfig().WithdrawalPrivkeyFileName, oldPassword, newPassword)
```
Comment (rauljordan) [inline validator/accounts/account.go:223]: +1

Comment (rauljordan) [inline validator/accounts/account.go:223]: You can just do `return changePasswordForKeyType(...`

File: validator/main.go [hunk 3]
```diff
@@ -24,6 +20,9 @@ import (
 	_ "go.uber.org/automaxprocs"
 	"gopkg.in/urfave/cli.v2"
 	"gopkg.in/urfave/cli.v2/altsrc"
+	"os"
+	"runtime"
```
Comment (0xKiwi) [inline validator/main.go:24]: Gotta run gofmt or goimports

File: validator/keymanager/direct_keystore.go [hunk 4]
```diff
@@ -2,6 +2,7 @@ package keymanager
 
 import (
 	"encoding/json"
+	"github.com/prysmaticlabs/prysm/shared/params"
```
Comment (0xKiwi) [inline validator/keymanager/direct_keystore.go:5]: Need goimports here too

**Pull request merged**